index.js

```diff
@@ -58,6 +59,11 @@ function handleArgs(cmd, args, opts) {
 		// #115
 		opts.cleanup = false;
 	}
+  
+  if (process.platform === 'win32' && (parsed.command === process.env.comspec || path.basename(parsed.command) === 'cmd.exe')) {
```

Comment:
Tried to resolve the conflict through mobile but it's a horrible UX... Feel free to fix it, otherwise will do it in the coming days.

Reply:
I didn't even know you could fix conflicts on mobile. I've fixed the stuff, no problem.

Reply:
Yes, but the editor is really horrible :)